tests/test_severity.py

```diff
@@ -0,0 +1,240 @@
+        }
+        self.critical_alert = {
+            'event': 'node_down',
+            'resource': 'node1',
+            'environment': 'Production',
+            'service': ['Network'],
+            'severity': 'critical',
+            'correlate': ['node_trace', 'node_down', 'node_marginal', 'node_up', 'node_pwned']
+        }
+        self.major_alert = {
+            'event': 'node_marginal',
+            'resource': 'node1',
+            'environment': 'Production',
+            'service': ['Network'],
+            'severity': 'major',
+            'correlate': ['node_trace', 'node_down', 'node_marginal', 'node_up', 'node_pwned']
+        }
+        self.warn_alert = {
+            'event': 'node_marginal',
+            'resource': 'node1',
+            'environment': 'Production',
+            'service': ['Network'],
+            'severity': 'warning',
+            'correlate': ['node_trace', 'node_down', 'node_marginal', 'node_up', 'node_pwned']
+        }
+        self.normal_alert = {
+            'event': 'node_up',
+            'resource': 'node1',
+            'environment': 'Production',
+            'service': ['Network'],
+            'severity': 'normal',
+            'correlate': ['node_trace', 'node_down', 'node_marginal', 'node_up', 'node_pwned']
+        }
+
+        self.trace_alert = {
+            'event': 'node_trace',
+            'resource': 'node1',
+            'environment': 'Production',
+            'service': ['Network'],
+            'severity': 'trace',
+            'correlate': ['node_trace', 'node_down', 'node_marginal', 'node_up', 'node_pwned']
+        }
+
+        self.ok_alert = {
+            'event': 'node_ok',
+            'resource': 'node2',
+            'environment': 'Production',
+            'service': ['Network'],
+            'severity': 'ok',
+            'correlate': []
+        }
+
+        self.inform_alert = {
+            'event': 'node_inform',
+            'resource': 'node3',
+            'environment': 'Production',
+            'service': ['Network'],
+            'severity': 'informational',
+            'correlate': []
+        }
+
+        self.headers = {
+            'Content-type': 'application/json'
+        }
+
+    def tearDown(self):
+
+        pass
+
+    def test_inactive(self):
+
+        # prevSev=unknown, sev=ok, status=closed, trend=noChange
+        response = self.app.post('/alert', data=json.dumps(self.ok_alert), headers=self.headers)
+        self.assertEqual(response.status_code, 201)
+        data = json.loads(response.data.decode('utf-8'))
+        self.assertEqual(data['alert']['duplicateCount'], 0)
+        self.assertEqual(data['alert']['repeat'], False)
+        self.assertEqual(data['alert']['previousSeverity'], 'unknown')
+        self.assertEqual(data['alert']['severity'], 'ok')
+        self.assertEqual(data['alert']['status'], 'closed')
+        self.assertEqual(data['alert']['trendIndication'], 'noChange')
```

Comment:
This is the test where the `trendIndication` is shown to be `noChange` for `ok` alerts received when previous severity is `unknown`.